content/hi/docs/concepts/architecture/cloud-controller.md

```diff
@@ -0,0 +1,215 @@
+---
+title: बादल नियंत्रक प्रबंधक
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+{{< feature-state state="beta" for_k8s_version="v1.11" >}}
+
+क्लाउड इंफ्रास्ट्रक्चर टेक्नोलॉजी आपको सार्वजनिक, निजी और हाइब्रिड क्लाउड पर कुबेरनेट चलाने देती है।
+कुबेरनेट्स स्वचालित, एपीआई-संचालित बुनियादी ढांचे के बीच तंग युग्मन के बिना विश्वास करता है
+अवयव।
+
+{{< glossary_definition term_id="cloud-controller-manager" length="all" prepend="The cloud-controller-manager is">}}
+
+क्लाउड-नियंत्रक-प्रबंधक को एक प्लगइन का उपयोग करके संरचित किया जाता है
+तंत्र जो विभिन्न क्लाउड प्रदाताओं को कुबेरनेट्स के साथ अपने प्लेटफॉर्म को एकीकृत करने की अनुमति देता है।
+
+<!-- body -->
+
+## डिज़ाइन
+
+![कुबेरनेट्स घटक](/images/docs/components-of-kubernetes.svg)
+
+क्लाउड कंट्रोलर मैनेजर कंट्रोल प्लेन में प्रक्रियाओं के एक प्रतिकृति सेट के रूप में चलता है
+(आमतौर पर, ये पॉड्स में कंटेनर होते हैं)। प्रत्येक क्लाउड-नियंत्रक-प्रबंधक लागू करता है
+एकाधिक {{< Glosary_tooltip text="controllers" term_id="controller" >}} एक में
```

Comment:
```suggestion
एकाधिक {{< glossary_tooltip text="controllers" term_id="controller" >}} एक में
```

In addition, you might localize the glossary_tooltip text `controllers` into Hindi as well.

```suggestion
एकाधिक {{< glossary_tooltip text="(Hindi word for 'controllers')" term_id="controller" >}} एक में
```